src/Symfony/Component/Cache/Tests/Adapter/AdapterTestCase.php

```diff
@@ -96,6 +98,18 @@ public function testPrune()
             $cache->save($item);
         };
 
+        $doSet('foo', 'foo-val', new \DateInterval('PT05S'));
```

Comment:
This may seem to duplicate tests, but the prior tests didn't check pruning of more than one item at a time; so this test ensures four items can be all pruned at once.